nin/backend/fontgen.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs-promise');
+const glob = require('glob');
+const path = require('path');
+
+
+async function fontGen(pathPrefix) {
+
+  let resolver;
+  let promise = new Promise(resolve => {
```

Comment:
https://github.com/sindresorhus/globby/blob/master/index.js#L67

Probably easier just to do it ourselves.